aiidalab_widgets_base/structures.py

```diff
@@ -437,7 +468,7 @@ def _on_select_structure(self, change):  # pylint: disable=unused-argument
         """When structure is selected."""
 
         self.structure = (
-            get_ase_from_file(self._select_structure.value)
+            get_ase_from_file(self._select_structure.value)[0]
```

Comment:
I am purposefully omitting any error handling here, since it's up to the programmer to provide valid example files.